client/flutter/lib/module_stores/ui/screen/store_list_screen.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:injectable/injectable.dart';
+import 'package:twaslna_delivery/module_stores/state_manager/store_list_state_manager.dart';
+import 'package:twaslna_delivery/module_stores/ui/state/store_list/store_list_loaded_state.dart';
+import 'package:twaslna_delivery/module_stores/ui/state/store_list/store_list_loading_state.dart';
+import 'package:twaslna_delivery/module_stores/ui/state/store_list/store_list_state.dart';
+import 'package:twaslna_delivery/utils/models/store_category.dart';
+
+@injectable
+class StoreListScreen extends StatefulWidget {
+  final StoreListStateManager _storeListManager;
+
+  StoreListScreen(this._storeListManager);
+
+  @override
+  StoreListScreenState createState() => StoreListScreenState();
+}
+
+class StoreListScreenState extends State<StoreListScreen> {
+  late StoreListState currentState;
+  bool flag = true;
```

Comment:
let us give this flag more clearly name that describes it's job

Reply:
Actually It's a flag and doing what flag do , It's very common name and have common use especially in OS programming (Semaphores)

Reply:
@Yazan99sh we do not use common. please resolve this so can make a merge.